tests/lightning_tensor/test_measurements_class.py

```diff
@@ -42,13 +36,25 @@
 
 # General LightningTensorNet fixture, for any number of wires.
 @pytest.fixture(
-    params=[np.complex64, np.complex128],
+    params=[
+        [c_dtype, method] for c_dtype in [np.complex64, np.complex128] for method in ["mps", "tn"]
+    ],
 )
 def lightning_tn(request):
     """Fixture for creating a LightningTensorNet object."""
 
     def _lightning_tn(n_wires):
-        return LightningTensorNet(num_wires=n_wires, max_bond_dim=128, c_dtype=request.param)
+        if request.param[1] == "mps":
```

Comment:
why not template the device kwargs as well?

Reply:
Done!